src/ROUTES.js

```diff
@@ -72,7 +72,10 @@ export default {
     getReportAttachmentRoute: (reportID, source) => `r/${reportID}/attachment?source=${encodeURI(source)}`,
     SELECT_YEAR: 'select-year',
     getYearSelectionRoute: (minYear, maxYear, currYear, backTo) => `select-year?min=${minYear}&max=${maxYear}&year=${currYear}&backTo=${backTo}`,
-
+    SETTINGS_SELECT_COUNTRY: 'select-country',
+    getCountrySelectionRoute: (countryISO, backTo) => `select-country?countryISO=${countryISO}&backTo=${backTo}`,
+    SETTINGS_USA_STATES: 'select-usa-states',
+    getUsaStateSelectionRoute: (stateISO, key, backTo) => `select-usa-states?stateISO=${stateISO}&key=${key}&backTo=${encodeURIComponent(backTo)}`,
```

Comment:
When using the `StatePicker` into the "connect bank manually" flow, it adds the URL a `policyID` -> `bank-account/?policyID=28B241A63D6EFC6F`, without encoding it that ID is lost and when going back it sends you to the wrong screen

Reply:
Ah I see. Is there any other option of passing the data besides the url parameters?

Reply:
I'm trying to use the [hook](https://github.com/Expensify/App/pull/21049#discussion_r1255573601) `useNavigationStorage`, but we would need to do more changes because I think the whole bank flow relays on it.

Reply:
Resolved.